addons/analytic/models/analytic_account.py

```diff
@@ -186,6 +187,7 @@ def _default_user(self):
     partner_id = fields.Many2one('res.partner', string='Partner', domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
     user_id = fields.Many2one('res.users', string='User', default=_default_user)
     tag_ids = fields.Many2many('account.analytic.tag', 'account_analytic_line_tag_rel', 'line_id', 'tag_id', string='Tags', copy=True, domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
+    # VFE FIXME shouldn't it be related to analytic account company_id ???????
```

Comment:
@smetl is it logical to have `account.analytic.line` having a required `company_id` (not related on their required account) whereas the one on the `account.analytic.account` isn't required ? :thinking: